packages/utilities/psammead-styles/CHANGELOG.md

```diff
@@ -3,23 +3,23 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
-| 1.0.1   | [PR#677](https://github.com/bbc/psammead/pull/677) Use `@bbc/gel-foundations@3.0.0` |
-| 1.0.0   | [PR#679](https://github.com/bbc/psammead/pull/679) Bump version number |
-| 0.6.0   | [PR#676](https://github.com/bbc/psammead/pull/676) Add Nassim Persian fonts v1.511 |
-| 0.5.0   | [PR#573](https://github.com/bbc/psammead/pull/573) Add functions to get font styles for different services |
-| 0.4.1   | [PR#547](https://github.com/bbc/psammead/pull/547) Create story for gel-foundations typographies |
-| 0.4.0   | [PR#548](https://github.com/bbc/psammead/pull/548) Bump the reith font version from 2.302 to 2.511 |
-| 0.3.5   | [PR#537](https://github.com/bbc/psammead/pull/537) Create story for colours |
-| 0.3.4   | [PR#505](https://github.com/bbc/psammead/pull/505) Add Metal; update Pebble |
-| 0.3.3   | [PR#424](https://github.com/bbc/psammead/pull/424) Add Snyk badge to readme |
-| 0.3.2   | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
-| 0.3.1   | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
-| 0.3.0   |  [PR#304](https://github.com/BBC-News/psammead/pull/304) Move font fallback definitions to gel-foundations, add font-face constants. |
-| 0.2.1   |  [PR#300](https://github.com/BBC-News/psammead/pull/300) Adds colours for Consent Banner |
-| 0.2.0   |  [PR#283](https://github.com/BBC-News/psammead/pull/283) Adds colours and fonts necessary for Simorgh V1.0 :art: |
-| 0.1.6   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
-| 0.1.5   | [PR#212](https://github.com/BBC-News/psammead/pull/212) Update package description and README. |
-| 0.1.4   | [PR#173](https://github.com/BBC-News/psammead/pull/173) Update PRs welcome link |
+| 1.0.1 | [PR#677](https://github.com/bbc/psammead/pull/677) Use `@bbc/gel-foundations@3.0.0` |
```

Comment:
Something's gone wrong here 🤔

Reply:
Could be that my PR to update gel-foundations omitted the version bump since `psammead-styles` is at 1.0.0 on latest

Not sure what the best solution here is, but maybe bump this package to 1.0.2?

Reply:
this package shoudn't have any changes, i'll double check what's going on here 😬